phpBB/phpbb/install/module_base.php

```diff
@@ -110,10 +110,38 @@ public function run()
 
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+		    // Receiving task index
```

Comment:
I'm not sure what you are referring to when you say `current()` being used in similar situations, if you could link me an example of where you seen it, it would be much appreciated. There are other ways to return to whatever point you'd like by setting simply setting the task index.

If you want to retrieve the previously submitted information you should look at the installer's config class. You can also see examples of it being used in other tasks.